src/main/java/org/broadinstitute/hellbender/tools/spark/sv/evidence/ExtractSVEvidenceSpark.java

```diff
@@ -0,0 +1,99 @@
+package org.broadinstitute.hellbender.tools.spark.sv.evidence;
+
+import htsjdk.samtools.SAMFileHeader;
+import org.apache.spark.api.java.JavaRDD;
+import org.apache.spark.api.java.JavaSparkContext;
+import org.apache.spark.broadcast.Broadcast;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.ArgumentCollection;
+import org.broadinstitute.barclay.argparser.BetaFeature;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.barclay.help.DocumentedFeature;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.cmdline.programgroups.StructuralVariantDiscoveryProgramGroup;
+import org.broadinstitute.hellbender.engine.spark.GATKSparkTool;
+import org.broadinstitute.hellbender.tools.spark.sv.StructuralVariationDiscoveryArgumentCollection;
+import org.broadinstitute.hellbender.tools.spark.utils.FlatMapGluer;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+import org.broadinstitute.hellbender.utils.read.SAMRecordToGATKReadAdapter;
+
+import static org.broadinstitute.hellbender.tools.spark.sv.evidence.FindBreakpointEvidenceSpark.buildMetadata;
+
+/**
+ * (Internal) Extracts evidence of structural variations from reads
+ *
+ * <p>This tool is used in development and should not be of interest to most researchers.  It repackages the first
+ * two steps of the structural variation workflow as a separate tool for the convenience of developers.</p>
+ * <p>This tool examines a SAM/BAM/CRAM for reads, or groups of reads, that demonstrate evidence of a structural
+ * variation in the vicinity.  It records this evidence as a group of text files in a specified output directory
+ * on Spark's HDFS file system.</p>
+ *
+ * <h3>Inputs</h3>
+ * <ul>
+ *     <li>A file of paired-end, aligned and coordinate-sorted reads.</li>
+ * </ul>
+ *
+ * <h3>Output</h3>
+ * <ul>
+ *     <li>A directory of text files describing the evidence for structural variation discovered.</li>
+ * </ul>
+ *
+ * <h3>Usage example</h3>
+ * <pre>
+ *   gatk FindBreakpointEvidenceSpark \
```

Comment:
Ha, ha.  Yes, of course it should.  Will fix.
W.r.t. minEvidenceMapQ:  The change is totally unimportant -- the two fields contain the same number, after all.  But if you'll allow me to wax briefly philosophical, referring the usage back to the original place where the value was defined, rather than to derived data, reduces the dependency tree (maybe you don't need a filter?), and also reduces the possibility of error introduced by refactoring (maybe the filter will alter the value somehow, someday?).  Redundant data is an enemy.  A datum defined in one place can be right or wrong, but it will never be inconsistent.  This, ever so slightly, reduces the dependency on redundant data.  OK.  Maybe that wasn't so brief.